contracts/core/Controller.sol

```diff
@@ -423,6 +422,15 @@ contract Controller is Initializable, OwnableUpgradeSafe, ReentrancyGuardUpgrade
         emit NakedCapUpdated(_collateral, _cap);
     }
 
+    /**
+     * @notice Sets margin pool v2
+     * @dev can only be called by the owner
+     * @param _marginPoolV2 margin pool v2 address
+     */
+    function setMarginPoolV2(address _marginPoolV2) external onlyOwner {
+        poolV2 = MarginPoolInterface(_marginPoolV2);
+    }
+
```

Comment:
This one can be removed, the `borrowablePool` address should be set in the AddressBook (following the current architecture) using the `AddressBook.setAddress()` function, then the address can be fetched and assigned to this variable in the `_refreshConfigInternal()`

```
borrowablePool = MarginPoolInterface(addressbook. getAddress(keccak256"BORROWABLE_POOL"));
```

Reply:
Ah, I didn't realize this was possible. Good solution!